RELEASE/scripts/autoscend/paths/g_lover.ash

```diff
@@ -48,33 +46,87 @@ boolean glover_usable(string it)
 	{
 		return true;
 	}
-	item checkItem = it.to_item();
-	if(checkItem != $item[none] && $items[SpinMaster&trade; lathe, // it works since there's no "use" link
+	return false;
+}
+
+boolean glover_usable(effect eff)
+{
+	if(!in_glover())
+	{
+		return true;
+	}
+	if($effects[Stone-Faced] contains eff)
+	{
+		return true;	//explicit exceptions that work in glover despite not having G in the name
+	}
+	return glover_usable(eff.to_string());
+}
+
```

Comment:
AFAIK, stone-faced is one of the only effects that is useful in G-Lover, and it can be Monkey Paw wished (although we don't do that yet and I haven't tried to add it yet either).